### PR TITLE
fix: fix the ellipsis bug for pre element

### LINKED ISSUE #48953: Typography.Paragraph ellipsis is not expandable if used inside <pre> for multiline text
### Reproduction link

[Edit on CodeSandbox](https://codesandbox.io/p/sandbox/antd-ellipsis-gkf7pp)

### Steps to reproduce

- create a <pre> element
- add a typography.paragraph inside with ellipsis: { rows: 2, expandable: true }
- render a long text inside the typography with multiple lines set by \n

### What is expected?

the text shows ellipsis after 2 lines and a button to expand the text

### What is actually happening?

ellipsis doesn't work

| Environment | Info |
| --- | --- |
| antd | 5.17.2 |
| React | react 18 |
| System | macos |
| Browser | chrome |

---
If the expandable is removed, then the text has the ellipsis but there's no way to expand it.

Not sure if this should even be a new component. Use case: from a Input.TextArea, allow the user to create a multiline text. Save that on DB, and when fetching and rendering, be able to render that text in multiline as well, but be able to use an expandable ellipsis if the content is too long

### PR DESCRIPTION
**Pull request by nova1751**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close #48953

### 💡 Background and solution

The css property `white-space: normal` will cause the `\n`  to not work in `pre` tag when the text is used as `MeasureText`. Removing it may be a good solution.🤔

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Fix the bug where the ellipsis option of `Typography` component doesn't work normally in `pre` tag        |
| 🇨🇳 Chinese |    修复 `Typography` 组件的 `ellipsis`选项在 `pre` 标签中没有正常工作的问题       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by nova1751** [comment]
Last PR was closed because I rebased the branch. So I opened a new one.

**Review by zombieJ — APPROVED** [review]

File: components/typography/Base/Ellipsis.tsx
Comment (hemengke1997) [inline components/typography/Base/Ellipsis.tsx:1]: 这里导致ellipsis测量错误了

**Comment by hemengke1997** [comment]
这个pr把 `whiteSpace: 'normal'` 删除后，导致某些情况下 `ellipsis` 功能失效了，我升级到此版本后发现的

**Comment by yoyo837** [comment]
> 这个pr把 `whiteSpace: 'normal'` 删除后，导致某些情况下 `ellipsis` 功能失效了，我升级到此版本后发现的

请打开一个新的issue.

**Pull request merged**